web/src/pm_list.js

```diff
@@ -160,8 +160,8 @@ export function handle_narrow_activated(filter) {
             `li[data-user-ids-string='${CSS.escape(current_user_ids_string)}']`,
         );
         scroll_pm_into_view($active_filter_li);
-        update_private_messages();
     }
+    update_private_messages();
 }
```

Comment:
This will rebuild the whole PM list everytime we narrow via `_build_direct_messages_list`. Is there something faster we can do here to reset the highlight?

Reply:
Let me look into it, If it can be done.